olot/utils/files.py

```diff
@@ -158,3 +159,45 @@ def handle_remove(path: os.PathLike):
         shutil.rmtree(path)
     else:
         os.remove(path)
+
+
+def walk_files_recursive(root_path: os.PathLike) -> List[Path]:
```

Comment:
```suggestion
def walk_files(root_path: os.PathLike) -> List[Path]:
```
nit - recursive vs iterative is an implementation detail (plus this isn't recursive)

Reply:
correct, was a leftover in naming, thanks :)